core/templates/dev/head/pages/exploration-editor-page/editor-tab/exploration-editor-tab.directive.ts

```diff
@@ -28,7 +28,17 @@ require(
 require(
   'pages/exploration-editor-page/editor-tab/unresolved-answers-overview/' +
   'unresolved-answers-overview.directive.ts');
+require(
+  'components/common-layout-directives/common-elements/' +
+  'loading-dots.directive.ts');
 require('components/state-editor/state-editor.directive.ts');
+require(
+  'components/state-directives/outcome-editor/' +
+  'outcome-destination-editor.directive.ts');
+require(
+  'components/state-directives/outcome-editor/' +
+  'outcome-feedback-editor.directive.ts');
+require('components/state-directives/rule-editor/rule-editor.directive.ts');
```

Comment:
Where are these three used?

Reply:
They are used in the core/templates/dev/head/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-answer-group-modal.template.html

Reply:
Ditto.

Reply:
Moved